pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -47,6 +47,8 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		Convert_v1beta1_SubresourceReference_To_autoscaling_CrossVersionObjectReference,
 		Convert_autoscaling_HorizontalPodAutoscalerSpec_To_v1beta1_HorizontalPodAutoscalerSpec,
 		Convert_v1beta1_HorizontalPodAutoscalerSpec_To_autoscaling_HorizontalPodAutoscalerSpec,
+		Convert_autoscaling_HorizontalPodAutoscalerStatus_To_v1beta1_HorizontalPodAutoscalerStatus,
+		Convert_v1beta1_HorizontalPodAutoscalerStatus_To_autoscaling_HorizontalPodAutoscalerStatus,
```

Comment:
no, those are populated by autoconvert, since the HPA object itself isn't different (it's just meta, spec, status in both cases, just the spec and status differ)